### PR TITLE
Widgets Preview: player.js refactor with paged elements grouping and effects

### PR DESCRIPTION
**Pull request by rubenpingol-xibo**
relates to xibosignageltd/xibo-private#365

### CODE REVIEW

File: modules/src/player.js [hunk 1]
```diff
@@ -469,159 +508,329 @@ $(function() {
...
+                              parentId: groupContentID,
+                              id: groupContentID,
+                            },
+                            $groupContentItem.find(`.${groupContentID}--item`),
+                          );
+                        });
+                      }
+                    }
+                  });
+                  // $.each(dataItems, function(_key, item) {
```
Comment (maurofmferrao) [inline modules/src/player.js:762]: Should we keep this commented code?

File: views/common.twig [hunk 2]
```diff
@@ -525,20 +525,8 @@
                     save: "{{ "Save" |trans }}",
                     cancel: "{{ "Cancel" |trans }}",
                 },
-                positionTab: {
```
Comment (maurofmferrao) [inline views/common.twig:528]: There was a merge conflict here, we need to keep the `positionTab` properties

**Review by maurofmferrao — CHANGES_REQUESTED** [review]
All looks good to me, except 2 minor changes

File: modules/src/player.js (hunk 1 above)
Comment (rubenpingol-xibo) [reply modules/src/player.js:762]: My bad. I'll have this removed.

File: views/common.twig (hunk 2 above)
Comment (rubenpingol-xibo) [reply views/common.twig:528]: Strange, I checked the copy of westphal and it doesn't have the `positionTab` in it. I'll have this added back.

**Review by maurofmferrao — APPROVED** [review]
Looks good!

**Pull request merged**